hyundai_kia_connect_api/KiaUvoApiCA.py

```diff
@@ -940,9 +944,9 @@ def set_charge_limits(
             "pin": token.pin,
         }
 
+        _LOGGER.debug(f"{DOMAIN} - Planned set_charge_limits payload {payload}")
```

Comment:
I am thinking we need to remove this or mask it. Since it will provide PINs in log files and users typically don't remove items like that.